app/experimenter/visualization/tests/api/test_views.py

```diff
@@ -2,14 +2,15 @@
 from unittest.mock import patch
 
 from django.conf import settings
-from django.test import TestCase
+from django.test import TestCase, override_settings
 from django.urls import reverse
 from parameterized import parameterized
 
 from experimenter.experiments.constants import ExperimentConstants
 from experimenter.experiments.tests.factories import NimbusExperimentFactory
 
 
+@override_settings(FEATURE_ANALYSIS=False)
```

Comment:
The visualization API code is referring to a setting being read from `.env`, so without overriding these tests break depending on what is set in your local dev, so this just isolates them.

Reply:
Also @emtwo see above I added some `__init__.py` to the visualization subtree that were missing.  These are so annoying, Python code will import them correctly but some things, like test discovery, will ignore them if they're missing.